CHANGELOG.md

```diff
@@ -1,3 +1,19 @@
+## 2.22.0-rc.0 / 2020-10-07
+
+* [CHANGE] web: Remove APIv2. #7935
```

Comment:
We should probably add a few more words here to avoid unnecessary worry, given that basically noone should be using this.

Reply:
Added.